_posts/2016-10-21-sweet-break-before-break-102.md

```diff
@@ -13,9 +13,9 @@ Thank Grohl, Gauss, Glob, Galileo and Science it’s Friday! This Friday’s lit
 
 Last week I hopped on a call with [Cecily](https://twitter.com/skeskali), [Gregor](https://twitter.com/gr2m), and [Jenn](https://twitter.com/jennwrites) to discuss the future of TGIF, Hoodie’s lazy Friday afternoon read started by [Lena](https://twitter.com/lrnrd) in [August 2013](http://hood.ie/blog/tgif-hoodies-top-links-of-the-week-1.html). After an hour-long chat with these amazing peeps we decided to take a step back and rethink how Hoodie’s weekly newsletter is created, delivered and promoted.
```

Comment:
I would update the timing on this since it’s been quite awhile since we all met.